apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -1305,6 +1305,22 @@ int wm_parse_commands(struct options *opt, int argc, char *argv[])
 		}
 		opt->func = wm_connect;
 		opt->ssid = argv[3];
+
+		if (argc == 5) {
+			opt->auth_type = get_auth_type(argv[4]);
```

Comment:
It might be better to remove the logic of handling auth_open in line 1325 - 1332 is now redundant. Please clean it up. Thanks.

Reply:
Fixed, I refined the logic.